third_party/js.bzl

```diff
@@ -312,6 +312,45 @@ def tensorboard_js_workspace():
           "https://github.com/pair-code/facets/archive/0.2.1.tar.gz",
       ],
   )
+  web_library_external(
+      name = "vaadin_vaadin_split_layout",
+      licenses = ["notice"],  # Apache License 2.0
+      sha256 = "44fb83628edb77cb8392c165d4d99734750a6fbb00e5391f033962e56f14eba3",
+      urls = [
+          "https://github.com/vaadin/vaadin-split-layout/archive/v1.1.0.tar.gz",
+          "https://mirror.bazel.build/github.com/vaadin/vaadin-split-layout/archive/v1.1.0.tar.gz",
```

Comment:
please reorder so the mirror link comes first. Here and below. AFAIK, web_library_external will fetch in order of declaration until it HTTP 2xxs.

Reply:
Done.